src/Relations/BelongsTo.php

```diff
@@ -72,4 +72,12 @@ protected function whereInMethod(EloquentModel $model, $key)
     {
         return 'whereIn';
     }
+
+    /**
+     * @return string
```

Comment:
CI supports php > 8.0 and master use illuminate packages so you can use return type hints and delete doc blocks

Reply:
@The-Hasanov can you do this?